### PR TITLE
Add separate effects for loading and showing blood pressure history

### PR DESCRIPTION
**Pull request by msasikanth**
https://app.shortcut.com/simpledotorg/story/7985/migrate-bloodpressurehistoryscreeneffecthandler-to-use-view-effects

### CODE REVIEW

**Comment by msasikanth** [comment]
Current dependencies on/for this PR:
* master
  * **PR #4153** <a href="https://app.graphite.dev/github/pr/simpledotorg/simple-android/4153" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/simpledotorg/simple-android/4153?utm_source=stack-comment).

**Comment by msasikanth** [comment]
Doing this in preparation for migrating the show blood pressures effect to view effect

**Review by janhavisinghh — APPROVED** [review]

**Comment by janhavisinghh** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/simpledotorg/simple-android/4153?utm_source=stack-comment).


Job Reference: IvJuIivk5tLQhvuWKVaV

**Pull request merged**
